aa-sdk/core/src/actions/smartAccount/dropAndReplaceUserOperation.ts

```diff
@@ -103,6 +103,8 @@ export async function dropAndReplaceUserOperation<
   // however, we have migrated to using erc7677middleware for alchemy paymaster flows
   // and most of the other paymasters we've seen don't do fee estimation
   const { maxFeePerGas, maxPriorityFeePerGas } = await resolveProperties(
+    // TODO(jh): this is not going to do anything if we are
```

Comment:
`dropAndReplacementUserOperation` won't work if gas required has gone up more than 10% since the original UO (i think this is probably the biggest issue w/ this PR and why i have it marked "experimental" for now)